js/modules.js

```diff
@@ -2,18 +2,90 @@ CORE.create_module("email-nav", function (sb) {
     return {
         init : function() {
             sb.onEvent(".email-nav_inbox", "click", function() {
+                $("#email-nav").find(".nav--selected").removeClass("nav--selected");
+                $(this).addClass("nav--selected");
                 sb.notify({
-                    type : "open-catalog"
+                    type : "open-category",
+                    data : "inbox"
```

Comment:
I believe ES6 allows you to have a trailing comma. Would be better to add a trailing comma so that any addition to this object doesn't touch the previous line. It's minor tho, I don't thing you need to make the change on this PR, we can just do it going forward